scripts/commands/checkOrphanPages.ts

```diff
@@ -0,0 +1,147 @@
+// This code is a Qiskit project.
+//
+// (C) Copyright IBM 2024.
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+import fs from "fs/promises";
+
+import { globby } from "globby";
+import yargs from "yargs/yargs";
+import { hideBin } from "yargs/helpers";
+import { flattenDeep } from "lodash";
+
+interface Arguments {
+  [x: string]: unknown;
+  currentApis: boolean;
+  devApis: boolean;
+  historicalApis: boolean;
+}
+
+const ALLOWED_ORPHANS = new Set([
+  "/api/qiskit/qiskit.primitives.BaseEstimator",
+  "/api/qiskit/qiskit.primitives.BaseSampler",
```

Comment:
CI is failing because `/run/circuit-execution` is not found.

Maybe call the constant `ALLOWED_ORPHAN_URLS` to make clear that they're not file paths, like not `docs/run/circuit-execution.ipynb`.